php/libraries/CouchDB.class.inc

```diff
@@ -194,9 +194,9 @@ class CouchDB
     function _constructURL($op, $doc)
     {
         $config = NDB_Config::singleton();
-        $db     = $config->getSetting("CouchDB");
+        $db     = $config->getSetting('CouchDB');
 
-        return "$op /$db[database]/$doc";
+        return "$op /$db[dbName]/$doc";
```

Comment:
I don't think the file has the basic tags need for couchDB, can you add those too